config/config-package.py

```diff
@@ -286,7 +292,18 @@ def copy_with_meta(template_name, destination, config_type, **kw):
             TomlArraySeparatorEncoderWithNewline(
                 separator=',\n   ', indent_first_line=True))
 
-    call(pathlib.Path(cwd) / 'bin' / 'tox', '-p', 'auto')
+    if args.no_linting:
+        old_toxenv = os.environ.get('TOXENV', None)
+        result = call(pathlib.Path(cwd) / 'bin' / 'tox', '-l',
+                      capture_output=True)
+        tox_envs = result.stdout.split()
+        if 'lint' in tox_envs:
+            tox_envs.remove('lint')
+        os.environ['TOXENV'] = ','.join(tox_envs)
+        call(pathlib.Path(cwd) / 'bin' / 'tox', '-p', 'auto')
+        os.environ['TOXENV'] = old_toxenv or ''
+    else:
+        call(pathlib.Path(cwd) / 'bin' / 'tox', '-p', 'auto')
```

Comment:
No, the flag should be passed over to the tox.ini.j2 template and used to skip the flake8/isort steps in [toxenv:lint].